src/pug/about.pug

```diff
@@ -34,11 +34,11 @@ block page
     div.box
       h2 Do you sell controllers?
       p Short answer: No.
-      p Long answer: Becoming a manufacturer and/or online shop requires a lot of work, money investment, certifications, legal preparations, and it is possibly much more boring that what we do now.
-      p So #[em maybe] in the future we could offer controller parts to buy directly from Input Labs website, though it depends on many factors.
+      p Long answer: For now we are only offering some selected parts, the more problematic for people to source or build. And even if we offer a bigger variety of DIY kits and parts later, we may never sell assembled controllers.
```

Comment:
=> "For now we are only offering some selected parts, which are  more problematic for people to source or build."